src/main/webapp/WEB-INF/views/drivers/add.jsp

```diff
@@ -23,10 +29,20 @@
         <td>
             <input type="text" name="license_number" form="driver" required>
         </td>
+        <td>
+            <input type="text" name="login" form="driver" required>
+        </td>
+        <td>
+            <input type="password" name="password" form="driver" required>
+        </td>
         <td>
             <input type="submit" name="add" form="driver">
         </td>
     </tr>
 </table>
+    <div class="footer">
+        <p>java web application with jsp and jdbc <a href="https://github.com/EliSoll">GitHub: Elvira Solnyshkina</a><br>
+            Image from <a href=" https://billionphotos.com">Billionphotos</a></p></div>
+</div>
```

Comment:
you could create a footer jsp and include it in each file where you need it

<%@include file="/WEB-INF/views/footer.jsp"%>